src/styles/blocks/stars.css

```
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  content: '';
+  background-image: url('../../images/star-active.svg');
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
```

```suggestion
  background-image: url('../../images/star-active.svg');
```
all those styles are redundant